sda-commons-server-spring-data-mongo/src/test/java/org/sdase/commons/server/spring/data/mongo/metrics/SpringDataMongoMetricsIT.java

```diff
@@ -106,6 +106,7 @@ void shouldSaveAndFind() {
     var tagsWithoutClusterId =
         commandsCompositeTime.get().getId().getTags().stream()
             .filter(t -> !"cluster.id".equals(t.getKey()))
+            .filter(t -> !"server.address".equals(t.getKey()))
```

Comment:
why do we need this additional line?

Reply:
I had one test execution where the server port matched `44` and the port is part of the `server.address`.